### ISSUE TITLE
NameError: name 'logging' is not defined

### ISSUE THREAD

**Issue by Gusa27** [glennake/DirectFire_Converter#8]
Hi Glenn, I am getting an error every time I try to run the script, even if I use the test ciscoasa_pre83.txt config:

`python3 converter.py -c /home/ubuntu/DirectFire_Converter/tests/ciscoasa_pre83/ciscoasa_pre83.txt -s "ciscoasa_pre83" -d "fortigate"`

```
Traceback (most recent call last):
  File "converter1.py", line 257, in <module>
    main(src_format=args.source, dst_format=args.destination, routing_info=args.routing)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/traceback_with_variables/print.py", line 98, in wrapper
    return func(*args, **kwargs)
  File "converter1.py", line 233, in main
    parsed_data = parse(
  File "converter1.py", line 107, in parse
    from DirectFire.Converter.parsers.ciscoasa_pre83 import parse
  File "/home/ubuntu/DirectFire_Converter/DirectFire/Converter/parsers/ciscoasa_pre83.py", line 23, in <module>
    logger = logging.getLogger(__name__)
NameError: name 'logging' is not defined
```

I am trying it from Ubuntu 20.04.2. 

pip freeze output:

```
ubuntu@ubuntu2004:~/DirectFire_Converter$ pip freeze
apturl==0.5.2
bcrypt==3.1.7
blinker==1.4
Brlapi==0.7.0
certifi==2019.11.28
chardet==3.0.4
Click==7.0
colorama==0.4.3
command-not-found==0.3
cryptography==2.8
cupshelpers==1.0
dbus-python==1.2.16
defer==1.0.6
distro==1.4.0
distro-info===0.23ubuntu1
duplicity==0.8.12.0
entrypoints==0.3
fasteners==0.14.1
future==0.18.2
httplib2==0.14.0
idna==2.8
keyring==18.0.1
language-selector==0.1
launchpadlib==1.10.13
lazr.restfulclient==0.14.2
lazr.uri==1.0.3
lockfile==0.12.2
logger==1.4
louis==3.12.0
macaroonbakery==1.3.1
Mako==1.1.0
MarkupSafe==1.1.0
monotonic==1.5
netifaces==0.10.4
oauthlib==3.1.0
olefile==0.46
paramiko==2.6.0
pexpect==4.6.0
Pillow==7.0.0
protobuf==3.6.1
pycairo==1.16.2
pycups==1.9.73
PyGObject==3.36.0
PyJWT==1.7.1
pymacaroons==0.13.0
PyNaCl==1.3.0
pyRFC3339==1.1
python-apt==2.0.0+ubuntu0.20.4.4
python-dateutil==2.7.3
python-debian===0.1.36ubuntu1
pytz==2019.3
pyxdg==0.26
PyYAML==5.3.1
reportlab==3.5.34
requests==2.22.0
requests-unixsocket==0.2.0
SecretStorage==2.3.1
simplejson==3.16.0
six==1.14.0
ssh-import-id==5.10
systemd-python==234
traceback-with-variables==2.0.1
ubuntu-advantage-tools==27.0
ubuntu-drivers-common==0.0.0
ufw==0.36
unattended-upgrades==0.1
Unidecode==1.2.0
urllib3==1.25.8
usb-creator==0.3.7
wadllib==1.3.3
xkit==0.0.0
```

Thanks.

**Comment by glennake** [comment]
Hi @Gusa27, 

Thanks for raising this, it should be fixed in latest commit 78b070f980ea8523cfa164fee3637c05cb3b1a4b.

Please pull and try again.

Thanks

**Comment by Gusa27** [comment]
Hi Glenn, 

It works now, thank you very much for the help.
